src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -63,6 +63,21 @@ void createConsumer(VertxTestContext context) throws InterruptedException, Timeo
             .deleteConsumer(context, groupId, name);
     }
 
+    @Test
+    void createConsumerAlreadyExists(VertxTestContext context) throws InterruptedException, TimeoutException, ExecutionException {
+        // create consumer
+        consumerService().createConsumer(context, groupId, consumerWithEarliestReset, HttpResponseStatus.OK);
+
+        // Create the same consumer
+        consumerService().createConsumer(context, groupId, consumerWithEarliestReset, HttpResponseStatus.CONFLICT);
+
+        context.completeNow();
```

Comment:
This has to be at the end of test. It set test as `PASS`, so if you perform any other action like `deleteConsumer()` it will not allow test result and I am sure, we don't want it. `assert` for context status after `context.completeNow()` is fine.

Reply:
Ok, changed the tst name to createDuplicatedConsumer